manifest.json

```diff
@@ -14,7 +14,8 @@
     "autoCloseUrlsConf": {
       "globalprotect": "gpcloudservice.com/SAML20/SP/ACS",
       "zoom": "zoom.us*success",
-      "sdm": "app.strongdm.com/app/auth/complete"
+      "sdm": "app.strongdm.com/app/auth/complete",
+      "notion native": "https://www.notion.so/native"
```

Comment:
i think coz i abuse the key as an element id it would have to be `notion_native`

Reply:
it seems to work, but I can change if you want.

Reply:
![image](https://user-images.githubusercontent.com/4451028/235743821-689d0ad1-3a9c-41a3-a2e2-265595e09c32.png)

Reply:
updated